Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityFocusProvider.cs

```diff
@@ -76,5 +86,19 @@ public interface IMixedRealityFocusProvider : IMixedRealitySourceStateHandler, I
         /// <typeparam name="T">The type of pointers to request. Use IMixedRealityPointer to access all pointers.</typeparam>
         /// <returns></returns>
         IEnumerable<T> GetPointers<T>() where T : class, IMixedRealityPointer;
+
+        /// <summary>
+        /// Subscribes to primary pointer changes.
+        /// </summary>
+        /// <param name="handler">Handler to be called when the primary pointer changes</param>
+        /// <param name="invokeHandlerWithCurrentPointer">Whether to invoke the handler with the current primary pointer 
```

Comment:
>invoke [](start = 69, length = 6)

Nit: Maybe add the clarification that this is sychronously invoking the handler (instead of just kicking off async work to immediately ping you back). It's possible that the "before subscribing" part is strong enough to describe this, but providing a little more detail here about the callback mechanics probably wouldn't hurt